src/js/utils/ContainerUtil.js

```diff
@@ -18,6 +18,19 @@ const ContainerUtil = {
    */
   adjustPendingActions(pendingActions, actionType, isPending) {
     return Object.assign({}, pendingActions, { [actionType]: isPending });
+  },
+
+  getNewContainerName(containerLength, newState) {
```

Comment:
`ContainerUtil.js` seemed like an appropriate spot for this, but I'm not sure. Can anybody recommend somewhere better?